readme.md

```diff
@@ -57,8 +57,7 @@ You must have Go > 1.6 installed.
     make build
 
 And to run hoverfly:
-
-    ./cmd/hoverfly/hoverfly
+    $GOPATH/bin/hoverflyfb
```

Comment:
Hi - docs are currently being rewritten. They should be in the next release. I'll be updating/improving the build instructions, so might leave this PR where it is for now... thanks though!